src/tree/tests/unit/Tree.spec.tsx

```diff
@@ -0,0 +1,659 @@
+const { describe, it, beforeEach } = intern.getInterface('bdd');
+const { assert } = intern.getPlugin('chai');
+import * as sinon from 'sinon';
+
+import { tsx } from '@dojo/framework/core/vdom';
+import { renderer, assertion, wrap } from '@dojo/framework/testing/renderer';
+import {
+	createMemoryResourceTemplate,
+	createResourceTemplate,
+	defaultFind
+} from '@dojo/framework/core/middleware/resources';
+
+import { Keys } from '../../../common/util';
+import { stubEvent } from '../../../common/tests/support/test-helpers';
+import Tree, { TreeNode, TreeNodeOption } from '../../index';
+import * as css from '../../../theme/default/tree.m.css';
+
+import { simpleTree } from './TreeData.mock';
+import LoadingIndicator from '../../../loading-indicator';
+
+const noop = () => {};
```

Comment:
this is exported from `common/tests/support/test-helpers`

Reply:
done 👍